apps-infra/infra-tools/__main__.py

```diff
@@ -0,0 +1,35 @@
+from pulumi_kubernetes.helm.v3 import Chart, ChartOpts, FetchOpts
+
+infra_apps = [
```

Comment:
You can use the Pulumi config plugin (similar to terraform vars) to insert such vars in a separated config.py file

Reply:
This a good start: https://www.pulumi.com/blog/pulumi-recommended-patterns-the-basics/